### PR TITLE
fix(Scripts/Items): Powerful Anti-venom

### PR DESCRIPTION
**Pull request by jestermaniac**
## Changes Proposed:

what it do.

missing script for the spell u dig. just what followed what was already done for lesser anti-venom shit.

on god

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7781
- Closes https://github.com/chromiecraft/chromiecraft/issues/1662

## How to Test the Changes:

1. Train First Aid to 300 to be able to use the item (.setskill 129 300)
2. .additem 19440 10
3. .aura 744 / 4940 / 6411 / 8313
4. Use item.
5. Observe poison status.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1631557649018772603.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1631557649018772603');
+
+DELETE FROM `spell_script_names` WHERE `spell_id` = 23786;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1631557649018772603.sql:3]:
```suggestion
DELETE FROM `spell_script_names` WHERE `spell_id` = 23786 AND `ScriptName` = 'spell_item_poweful_anti_venom';
```

**Review by Kitzunu — DISMISSED** [review]

**Comment by IntelligentQuantum** [comment]
Tested

Screenshot_20210914_091118

File: src/server/scripts/Spells/spell_item.cpp [hunk 2]
```diff
@@ -865,6 +865,52 @@ class spell_item_gnomish_universal_remote : public SpellScriptLoader
...
+            PreventHitDefaultEffect(effIndex);
+            if (Unit* target = GetHitUnit())
+            {
+                std::list<uint32> removeList;
+                Unit::AuraMap const& auras = target->GetOwnedAuras();
+                for (Unit::AuraMap::const_iterator itr = auras.begin(); itr != auras.end(); ++itr)
+                {
+                    Aura* aura = itr->second;
+                    if (aura->GetSpellInfo()->SpellLevel > 60 || aura->GetSpellInfo()->Dispel != DISPEL_POISON)
+                    {
```
Comment (patou01) [inline src/server/scripts/Spells/spell_item.cpp:888]: Could it make more sense to use the following snippet?  Seems easier to read than a negative, but maybe it breaks some guidelines that I have not seen.
```
if(aura->GetSpellInfo()->SpellLevel <= 60 && aura->GetSpellInfo()->Dispel == DISPEL_POISON)
{
    removeList.push_back(aura->GetId());
}
```

Comment (UltraNix) [reply src/server/scripts/Spells/spell_item.cpp:888]: No difference.

**Review by UltraNix — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
@IntelligentQuantum @UltraNix please don't forget to apply the workflow labels so we avoid PRs to be forgotten

**Comment by Helias** [comment]
related to: https://github.com/azerothcore/azerothcore-wotlk/pull/4711
they are identical, the only change is the level of the poison "25" and "60", we should refactor this in the next future and make someting re-usable

probably even anti-venom "strong" is the same

**Pull request merged**